pkg/utils/test/oc/get.go

```diff
@@ -0,0 +1,117 @@
+package oc
```

Comment:
@vimalk78 , we need license on all our files :(

Reply:
adding

Reply:
old files also do not have license

Reply:
We need a checker in CI to do these mundane checks 🤯 ...